flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/source/meta/offset/Offset.java

```diff
@@ -39,7 +39,8 @@ public abstract class Offset implements Comparable<Offset>, Serializable {
 
     protected Map<String, String> offset;
```

Comment:
If the return type is changed , this field should be changed too.

Reply:
other connectors rely on the string value type/

e.g.,  if we change the filed, for oracle, it will have errors like this

> RedoLogOffset.java:57:26
java: incompatible types: capture#1 of ? cannot be converted to java.lang.String

Reply:
How about changing like this.
Offset.java
```java
protected Map<String, ?> offset;

public Map<String, ?> getOffset() {
        return offset;
    }
```

Then all its subclass need to cast the value from this map in usage. 
For example : RedoLogOffset.java
```java
public String getScn() {
        return (String) offset.get(SCN_KEY);
    }

    public String getCommitScn() {
        return (String) offset.get(COMMIT_SCN_KEY);
    }

    public String getLcrPosition() {
        return (String) offset.get(LCR_POSITION_KEY);
    }
```

Reply:
refactored to use String value type and only convert map when loading offset